File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/Preview.java
package io.github.gusandrianos.foxforreddit.data.models;

import java.util.List;
import com.google.gson.annotations.SerializedName;


public class Preview{
    @SerializedName("enabled")
    private boolean enabled;

    @SerializedName("reddit_video_preview")
    private RedditVideoPreview redditVideoPreview;

    public RedditVideoPreview getRedditVideoPreview() {
        return redditVideoPreview;
    }
}

File: app/src/main/java/io/github/gusandrianos/foxforreddit/data/models/Preview.kt
package io.github.gusandrianos.foxforreddit.data.models

import com.google.gson.annotations.SerializedName

class Preview {
    @SerializedName("enabled")
    val enabled = false

    @SerializedName("images")
    val images: List<ImagesItem>? = null

    @SerializedName("reddit_video_preview")
    val redditVideoPreview: RedditVideoPreview? = null
}